listeners.py

```diff
@@ -33,13 +34,19 @@ def process_timeout(self, delta_seconds):
 class StatisticsWatcher(BaseWatcher):
     def __init__(self, get_logfile):
         self.stats = dict()
+        self.rss_results = sampler.rss_summary()
         self.failed_tasks = []
         self.get_logfile = get_logfile
+        self.long_tasks = []
 
     def process_result(self, obj):
         if not isinstance(obj, WorkerTaskResult):
             return
 
+        task_name = obj.task_id[0] + ((':' + obj.task_id[1]) if obj.task_id[1] else '')
+        if obj.is_long:
+            self.long_tasks.append(task_name)
```

Comment:
Nit: I would prefer to use `task_id` (a tuple) in internal structures rather than the human readable test name.

Reply:
Ok, done.